docker/uwsgi.ini

```diff
@@ -17,10 +17,10 @@ enable-threads  = true
 
 # Run watchmedo (via watchdog) to implement unrestricted
 # autoreload capability.
-fs-reload = %(base)/docker/artifacts/
-safe-pidfile = %(base)/docker/artifacts/addons-server-uwsgi-master.pid
-attach-daemon = setsid watchmedo shell-command --patterns="*.py" --recursive --command='/usr/bin/touch %(safe-pidfile)' %(base)
-max-requests = 5000
+# fs-reload = %(base)/docker/artifacts/
+# safe-pidfile = %(base)/docker/artifacts/addons-server-uwsgi-master.pid
+# attach-daemon = setsid watchmedo shell-command --patterns="*.py" --recursive --command='/usr/bin/touch %(safe-pidfile)' %(base)
+# max-requests = 5000
```

Comment:
I think we want this for local env, so you can't disable it generally

Reply:
I'd probably try to use https://uwsgi-docs.readthedocs.io/en/latest/ConfigLogic.html?highlight=if%20env#if-env and only set `safe-pidfile` and `attach-daemon` in case of that environment variable exists. Or, do the reverse, do something like

```ini
# MOZ_HEADLESS is only being set in docker-compose.selenium.yml which is only used by circleci
if-env = MOZ_HEADLESS
attach-daemon =
endif =
```

or so… this is untested, I don't know if unsetting variables like that works.

Reply:
Since this only effects CI why don't we use a circleci env variable. https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables, so ```if-not-env = CI```